src/Entity/Geometry.php

```diff
@@ -293,6 +293,12 @@ public function getGeometryType(): ?string
         return $this->geometryType;
     }
 
+    public function getGeometryTypes(): array
+    {
+        preg_match_all('/\d+/', $this->getGeometryType() ?? 0, $matches);
+        return array_map('intval', $matches[0] ?? []);
```

Comment:
just a way to extract the numbers from the comma separated string . 
Assumes non-negative numbers

Reply:
should be fine, types should always be ints of 0 or higher

Reply:
although couldn't you just use 
`explode(',', $this->getGeometryType());`